### PR TITLE
Adds ability for user to compare query results from two countries

### PR DESCRIPTION
**Pull request by mantagen**
image

### CODE REVIEW

File: client/src/Update/Update.elm [hunk 1]
```diff
@@ -286,14 +314,14 @@ update msg model =
                 ( { model | categorySelect = updatedCategorySelectModel }, Cmd.none )
 
         FetchQueryResults (Ok results) ->
-            ( { model | queryResults = results }, Cmd.none )
+            ( { model | queryResults = results.results }, Cmd.none )
```
Comment (Jbarget) [inline client/src/Update/Update.elm]: did Chris change the API?

File: client/src/QueryResultList.elm [hunk 2]
```diff
@@ -1,30 +1,62 @@
...
+                        , onClick (QueryResultListRemoveClick index)
+                        , ariaLabel ("Remove " ++ countryName ++ " from comparison")
+                        ]
+                        []
+                    )
                 ]
-                []
             , ul [ class "list f5" ]
                 (List.indexedMap
                     (\index match ->
```
Comment (Jbarget) [inline client/src/QueryResultList.elm]: you can remove `pointer` from the `accordionToggleClass` in this file

Comment (Jbarget) [reply client/src/QueryResultList.elm]: also `submenuButtonClass` in `SelectedCategories.elm`

Comment (Jbarget) [reply client/src/QueryResultList.elm]: and line 49 of `SelectedCategories`

File: client/src/Helpers/QueryString.elm [hunk 3]
```diff
@@ -20,4 +35,60 @@ queryString model =
...
-        countries ++ activity ++ categories ++ filterText
+        countries ++ countries2 ++ activity ++ categories ++ filterText
+
+
+removeFromSearchParsed : ( String, Int ) -> SearchParsed -> SearchParsed
+removeFromSearchParsed ( key, i ) searchParsed =
+    case Dict.get key searchParsed of
+        Just oldParams ->
+            Dict.update key
+                (\mv ->
```
[the remark this answers is not in the document]
Comment (mantagen) [reply client/src/Helpers/QueryString.elm]: maybe value

File: client/src/Update/Update.elm (hunk 1 above)
Comment (mantagen) [reply client/src/Update/Update.elm]: no i changed our node route for multiple countries

File: client/src/QueryResultList.elm (hunk 2 above)
Comment (mantagen) [reply client/src/QueryResultList.elm]: :+1:

**Pull request merged**